src/main/java/by/custom/utilcalculator/domain/tree/HelperTree.java

```diff
@@ -0,0 +1,69 @@
+package by.custom.utilcalculator.domain.tree;
+
+import by.custom.utilcalculator.domain.constants.Command;
+import by.custom.utilcalculator.domain.constants.steps.*;
+
+import java.util.*;
+
+public class HelperTree {
+    public static Map<StepsIndicator, String> fillMap() {
```

Comment:
Which map to fill? It's too abstract. I suggest `fillFieldsToCommandsMap`.

Reply:
Renamed, thanks